tester_app/client/bootstrap-sass.config.js

```diff
@@ -0,0 +1,88 @@
+// IMPORTANT: Make sure to keep the customizations defined in this file
+//            in-sync with the ones defined in app/assets/stylesheets/_bootstrap-custom.scss.
+
+// For a reference on customizations,refer to:
+//  https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss
+
+module.exports = {
+  reactOnRailsSassHelper: './assets/stylesheets/_react-on-rails-sass-helper.scss',
```

Comment:
that doesn't work

there's only one pre and post file

you need to reference this in the pre file with an import

please be sure to try all versions of the apps

look for errors and other anomolies